internal/provider/data_source_packer_image_iteration.go

```diff
@@ -0,0 +1,224 @@
+package provider
+
+import (
+	"context"
+	"log"
+	"time"
+
+	packermodels "github.com/hashicorp/hcp-sdk-go/clients/cloud-packer-service/preview/2021-04-30/models"
+	sharedmodels "github.com/hashicorp/hcp-sdk-go/clients/cloud-shared/v1/models"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/hashicorp/terraform-provider-hcp/internal/clients"
+)
+
+var defaultPackerTimeout = time.Minute
+
+func dataSourcePackerImageIteration() *schema.Resource {
+	return &schema.Resource{
+		Description: "The Packer Image data source iteration gets the most recent iteration (or build) of an image, given a channel.",
+		ReadContext: dataSourcePackerImageRead,
+		Timeouts: &schema.ResourceTimeout{
+			Default: &defaultPackerTimeout,
+		},
+		Schema: map[string]*schema.Schema{
+			// Required inputs
+			"bucket": {
```

Comment:
Are there any plans to make `bucket` and `channel` resources so this can all be managed by TF?